module2/success/learning_goals.md

```diff
@@ -4,7 +4,7 @@ title: Module 2
 subheading: Learning Goals
 ---
 
-In addition to our [guidlines and expectations](./guidelines_and_expectations), academic success in Module 2 means that students demonstrate proficiency and comfort with the concepts below.
+Academic success in Module 2 means that students demonstrate proficiency and comfort with the concepts below.
```

Comment:
I decided to take out this "Guidelines and Expectations" page. You can view the original page (here)[https://backend.turing.io/module2/success/guidelines_and_expectations]. I like everything that is said in that page, but I feel like it is not appropriate for this to be a mod 2 specific page. For example, the section about "Attendance" gets outdated as we update the school-wide attendance policy. We could consider rolling this info into our intro to mod 2 slides if we think there's anything from this Guidelines and Expectations doc we'd like to keep.

Reply:
That makes sense to me.